docs/validation.md

```diff
@@ -116,6 +116,25 @@ matchbox:
           - "regex:Binding for path (.+).ofType\(Coding\) has no source, so can't be checked"
 ```
 
+#### Suppress error issues in validation {: #suppress-errors}
+
+The validation client can suppress error issues that are not relevant for the validation.
+Validation issues to be suppressed are defined in the configuration file, per Implementation Guide and version.
+
+You need to provide the path in the resource and the [message](https://github.com/hapifhir/org.hl7.fhir.core/blob/master/org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/i18n/I18nConstants.java) where the error is suprressed. The path (optional) and messageId neets to be concatenated by a the ! charachter, e.g: "path!messageId". If the mesasgeId should be suppressed on any path, use the charachter * as path.
+
+Example of configuration file:
+```yaml
+matchbox:
+  fhir:
+    context:
+      suppressError:
+        hl7.fhir.r4.core#4.0.1:
+          - "RelatedPerson!Extension_EXTP_Context_Wrong"
```

Comment:
The path is a FHIRPath? Can we have another example to make it clear (like `Patient.identifier`)

Reply:
`OperationOutcome.issue.expression` [is a FHIRPath](https://hl7.org/fhir/R4/operationoutcome-definitions.html#OperationOutcome.issue.expression) (or rather a limited subset of FHIRPath), although in that example it doesn't seem valid.
A regex is probably a good idea here, since the indexes and ids could easily change.